docker/Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM mongo:latest
 
+# GHCR labels
+ARG SOURCE_REPO="https://github.com/Sage-Bionetworks/agora"
```

Comment:
to make it more reusable for other repos consider using:
`ARG SOURCE_REPO="https://github.com/${{ github.repository }}"`

Reply:
Agreed on reusability! I've tried to do that by making `SOURCE_REPO` a build-time [ARG](https://docs.docker.com/reference/dockerfile/#arg), so it can be overwritten when the Docker image is built. The current value is a default that primarily demonstrates what `SOURCE_REPO` should look like. 

In the GHA workflow, we override the value at build-time [here](https://github.com/Sage-Bionetworks/Agora/blob/48b4d5a33892756af126c13e8838486cf9c9a87d/.github/workflows/e2e.yml#L138), but I agree that using `${{ github.repository }}` would be clearer, so I'll update to use that instead!